lib/amazonka/src/Amazonka/Auth/ConfigFile.hs

```diff
@@ -253,21 +302,39 @@ data CredentialSource = Environment | Ec2InstanceMetadata | EcsContainer
 --
 -- Throws 'MissingFileError' if 'credFile' is missing, or 'InvalidFileError'
 -- if an error occurs during parsing.
+
+-- If @AWS_SHARED_CREDENTIALS_FILE@ is set, it will be used instead of looking
+-- for @.aws\/credentials@ in the @HOME@ directory
+-- If @AWS_CONFIG_FILE@ is set, it will be used instead of looking for
+-- @.aws\/config@ in the @HOME@ directory.
+-- If @AWS_PROFILE@ is set, it will be used instead of the default profile
 --
 -- This looks in in the @HOME@ directory as determined by the
 -- <http://hackage.haskell.org/package/directory directory> library.
 --
+
 -- * Not Windows: @$HOME\/.aws\/credentials@
+
 --
+
 -- * Windows: @%USERPROFILE%\\.aws\\credentials@
+
```

Comment:
Please remove the blank lines in this section.

Reply:
huh idk where these came from. ormolu maybe went rogue